anthos-bm-gcp-terraform/README.md

```diff
@@ -0,0 +1,162 @@
+################################################################################
+
+# ------------------------------------------------------------------------------
+# SSH into it the admin host
+# ------------------------------------------------------------------------------
+gcloud compute ssh root@sabm-ws-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+
+# ------------------------------------------------------------------------------
+# Use must be SSH'ed into the admin host sabm-ws-001 as root user now
+# ------------------------------------------------------------------------------
+bmctl create config -c dev-abm-cluster && \
+cp ~/dev-abm-cluster.yaml bmctl-workspace/dev-abm-cluster && \
+bmctl create cluster -c dev-abm-cluster
+################################################################################
+```
+***The above should setup the baremetal cluster. This includes doing preflight checks on the nodes, creating the admin and user clusters and also registering the cluster with Google Cloud using [Connect](https://cloud.google.com/anthos/multicluster-management/connect/overview). The whole setup may take upto approx. 15 minutes***
+
+You will see the following output as the ***admin cluster*** is being created
+```sh
+Created config: bmctl-workspace/dev-abm-cluster/dev-abm-cluster.yaml
+Creating bootstrap cluster... OK
+Installing dependency components... OK
+Waiting for preflight check job to finish... OK
+- Validation Category: machines and network
+        - [PASSED] 10.200.0.3
+        - [PASSED] 10.200.0.4
+        - [PASSED] 10.200.0.5
+        - [PASSED] 10.200.0.6
+        - [PASSED] 10.200.0.7
+        - [PASSED] gcp
+        - [PASSED] node-network
+Flushing logs... OK
+Applying resources for new cluster
+Waiting for cluster to become ready OK
+Writing kubeconfig file
+kubeconfig of created cluster is at bmctl-workspace/dev-abm-cluster/dev-abm-cluster-kubeconfig, please run
+kubectl --kubeconfig bmctl-workspace/dev-abm-cluster/dev-abm-cluster-kubeconfig get nodes
+to get cluster node status.
+Please restrict access to this file as it contains authentication credentials of your cluster.
+Waiting for node pools to become ready OK
+Moving admin cluster resources to the created admin cluster
+Flushing logs... OK
+Deleting bootstrap cluster... OK
+```
+
+---
+### Verify and interacting with the Baremetal cluster
+
+You can find your cluster's `kubeconfig` file on the admin machine in the `bmctl-workspace` directory. To verify your deployment, complete the following steps
+
+1. SSH into the admin workstation _(if you are not already inside it)_:
+```sh
+> # You can copy the command from the output of terraform run from the previous step
+> gcloud compute ssh root@abm-ws-001 --project=<YOUR_PROJECT> --zone=<YOUR_ZONE>
+```
+
+2. Set the `KUBECONFIG` environment variable with the path to the cluster's configuration file to run `kubectl` commands on the cluster.
+```sh
+> export CLUSTER_ID=anthos-gce-cluster
```

Comment:
$CLUSTER_ID doesn't match the terraform.tfvars.sample abm_cluster_id

Reply:
Fixed in d9b96ce8ba60f1f687ec7a0a318b55e6977c7b0f